### ISSUE TITLE
Flashlist seems like overrides the allowHeaderOverscroll to true  if there isn't enough content to fill the screen

### ISSUE THREAD

**Issue by MtyldZ** [PedroBern/react-native-collapsible-tab-view#431]
Hi, I noticed this issue when I was changing my tabs written with react-native-tab-view to this package. It seems that when you try to refresh flashlist refresh control appears at the top (just like setting allowHeaderOverscroll to true) , when there isn't enough items within the flashlist. I modified the example project and get the same results. Here are the differences; 

* example/src/FlashList.tsx:
 comment out allowHeaderOverscroll prop
* example/src/Shared/ExampleComponentFlashList.tsx:
 replace `<ContactsFlatList emptyContacts={emptyContacts} limit={5} />` with `<ContactsFlashList emptyContacts={emptyContacts} limit={5} />`. (update the last tab to have a flashlist with less items)
 
 

https://github.com/PedroBern/react-native-collapsible-tab-view/assets/56448237/bd0e5003-0136-4d82-92cf-ef3c1681f831

**Comment by jinpark0625** [comment]
any updates?

**Comment by MtyldZ** [comment]
I don't believe there has been any update. I had to use regular Flatlist for my case.

**Comment by andreialecu** [comment]
This is a duplicate of https://github.com/PedroBern/react-native-collapsible-tab-view/issues/335

It's an upstream limitation with FlashList.
